keras_cv/bounding_box/to_ragged.py

```diff
@@ -49,6 +50,12 @@ def to_ragged(bounding_boxes, sentinel=-1, dtype=tf.float32):
         dictionary of `tf.RaggedTensor` or 'tf.Tensor' containing the filtered
         bounding boxes.
     """
+    if backend.supports_ragged() is False:
+        raise NotImplementedError(
+            "`bounding_box.to_ragged` was called using a backend which does "
```

Comment:
Always good to specify which backend in an error msg like this

Reply:
Done